source/pug/partials/events.pug

```diff
@@ -4,7 +4,9 @@
     h4 Launch events
 
 .container
-  include:markdown ../../markdown/events.md
+  .row.justify-content-center
+    .col-12.col-xl-10
```

Comment:
Gotcha. Maybe we should use`.col-lg-10` instead of `.col-12.col-xl-10` so max-width doesn't exceed 920px.